tldw_Server_API/app/core/MCP_unified/tests/test_slides_module.py

```
@@ -200,4 +257,152 @@ async def _fake_rag_pipeline(**kwargs):
+    mod = SlidesModule(ModuleConfig(name="slides"))
+    fake_db = FakeSlidesDB()
+    mod._open_db = lambda ctx: fake_db  # type: ignore[attr-defined]
+    ctx = SimpleNamespace(db_paths={"slides": "unused.db"})
+
+    created = await mod.execute_tool(
+        "slides.presentations.create",
+        {"title": "Deck", "slides": json.dumps({"slides": [{"order": 0}, {"order": 1}]})},
+        context=ctx,
+    )
+
+    with pytest.raises(ValueError, match="slide_order must be a permutation of 0..1"):
```

Ruff (0.15.9)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)